### PR TITLE
Kafka Node Pools: What's next?

### PR DESCRIPTION
**Pull request by scholzj**
This Pr delivered the last blog post for the Node Pools series called _What's next?_. It covers some of the next steps and future improvements that might happen.

_(It will be published only after the fourth part from #382)_

### Type of change

* [x] New blog post (see the [README](https://github.com/strimzi/strimzi.github.io/#blog-posts) for the process)

### CODE REVIEW

File: _posts/2023-08-31-kafka-node-pools-whats-next.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+This all can be done while maintaining the availability of the Kafka cluster and without the need to have two clusters and use mirroring to migrate between them.
+
+The stretch cluster feature is not something you should expect anytime soon.
+We might look into it at some point next year.
+
+### Conclusion
+
+This is the last part of this blog post series.
+Hopefully, over the five parts we managed to show you how to use the node pools and explain some of the motivations behind them.
+Give it a try and we will be looking forward to your feedback, raised bugs, feature requests, or PRs.
```
Comment (PaulRMellor) [inline _posts/2023-08-31-kafka-node-pools-whats-next.md:126]:
```suggestion
Do give them a try.
And don't hesitate to get in touch. 
We look forward to hearing your feedback and addressing any issues you encounter.
```

**Review by PaulRMellor — APPROVED** [review]
Great series. I left a few suggestions.

**Review by see-quick — APPROVED** [review]
Very well written. LGTM 👍

File: _posts/2023-08-31-kafka-node-pools-whats-next.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+Already today, if you want to try the KRaft mode, you have to do it using the `KafkaNodePool` resources.
+As Apache Kafka completes the work on the KRaft mode and completely drops support for ZooKeeper, everyone will need to migrate from ZooKeeper-based to KRaft-based Kafka clusters.
+And as part of this migration, all Strimzi users will also move to use the `KafkaNodePool` resources.
+
+#### Strimzi `v1` APIs
+
+Once the migration to KRaft and node pools moves forward, we will also start working on the Strimzi `v1` API.
+Evolving the APIs of the Custom Resources can be complicated.
+However, once the ZooKeeper support is removed from Apache Kafka and Strimzi, we will be able to remove the ZooKeeper-related options from the `Kafka` custom resource definition.
+We will be also able to remove the Kafka options that moved into the `KafkaNodePool` resources such as the number of replicas or storage configuration.
```
Comment (tombentley) [inline _posts/2023-08-31-kafka-node-pools-whats-next.md:48]: This sentence was a bit confusing. 

```suggestion
We will be also able to remove from the `Kafka` resource those options that have moved into the `KafkaNodePool` resources, such as the number of replicas or storage configuration.
```

File: _posts/2023-08-31-kafka-node-pools-whats-next.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+* Templates for customizing the resources belonging to this pool, such as pods or containers
+
+This may be expanded in the future. 
+Certain options, like cluster-wide configurations for authorization or listeners, wouldn't typically be configured at the node pool level. 
+However, other options can easily be added to the configuration at the node pool level if they offer value.
+
+Would there be some use case to configure a different container image for each node pool?
+Different metrics or logging configurations?
+Tuning some of the Kafka configuration options that can be applied on a per-node level?
+If you have a use case in mind for any of these options, feel free to let us know through GitHub Discussions or Issues and we can take a closer look.
```
Comment (tombentley) [inline _posts/2023-08-31-kafka-node-pools-whats-next.md:67]: Discussions and Issues as links?

**Review by tombentley — APPROVED** [review]
LGTM!

**Review by ppatierno — APPROVED** [review]

**Pull request merged**